packages/@tinacms/cli/src/cmds/audit/index.ts

```diff
@@ -23,10 +23,19 @@ import { auditCollection, auditDocuments } from './audit'
 import { logger } from '../../logger'
 import chalk from 'chalk'
 import prompts from 'prompts'
+import { Telemetry } from '@tinacms/metrics'
 
 const rootPath = process.cwd()
 
 export const audit = async (ctx: any, next: () => void, options) => {
+  const t = new Telemetry({ disabled: options.noTelemetry })
```

Comment:
nitpick - a better variable name would be helpful here (and for the other sections like this).

Reply:
Changed!